airflow/settings.py

```diff
@@ -162,9 +164,16 @@ def configure_orm(disable_connection_pool=False):
 try:
     from airflow_local_settings import *
     logging.info("Loaded airflow_local_settings.")
-except:
+except Exception:
     pass
 
+if 'AirflowTaskLogging' not in sys.modules:
+    # No user-defined task logging setting loaded in airflow_local_settings.
+    # Import default airflow logging configurations.
+    from airflow.utils.log.airflow_task_logging import AirflowTaskLogging
```

Comment:
Let's log debug here whether or not the user defined airflow task logging was loaded.